open-metadata-resources/open-metadata-archives/design-model-archives/src/main/java/org/odpi/openmetadata/archiveutilities/designmodels/base/DesignModelArchiveBuilder.java

```diff
@@ -754,6 +835,70 @@ protected void addLinkToBead(String                 beadId,
                                                                      end2));
     }
 
+    /**
+     * Add an is a relationship
+     * @param specialTermQName qualified name of the specialized term
+     * @param generalizedTermQName qualified name of the generalized term
+     */
+    protected void addISARelationship (String specialTermQName , String generalizedTermQName) {
+
+        String specializedTermId = idToGUIDMap.getGUID(specialTermQName);
+        String generalizededTermId = idToGUIDMap.getGUID(generalizedTermQName);
+
+
+        EntityProxy end1 = archiveHelper.getEntityProxy(archiveBuilder.getEntity(generalizededTermId));
+        EntityProxy end2 = archiveHelper.getEntityProxy(archiveBuilder.getEntity(specializedTermId));
+
+        archiveBuilder.addRelationship(archiveHelper.getRelationship(ISA_RELATIONSHIP_NAME,
+                                                                     idToGUIDMap.getGUID(generalizededTermId + "_to_" + specializedTermId),
+                                                                     null,
+                                                                     InstanceStatus.ACTIVE,
+                                                                     end1,
+                                                                     end2));
+    }
+    protected void addHasARelationship(String conceptQName, String propertyQName) {
+        String conceptId = idToGUIDMap.getGUID(conceptQName);
```

Comment:
Formatting - problem continues futher down file

Reply:
I think you are looking at a back level fix - this is not in the current fix